### PR TITLE
Extra Info message

### PR DESCRIPTION
**Pull request by wouter-heerwegh**
Post extra information through the ros bridge like the doo_counter and the amount of laps.

### CODE REVIEW

File: ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp [hunk 1]
```diff
@@ -818,6 +820,24 @@ void AirsimROSWrapper::finished_signal_cb(fs_msgs::FinishedSignalConstPtr msg)
...
+    } catch (rpc::rpc_error& e)
+    {
+        std::string msg = e.get_error().as<std::string>();
+        std::cout << "Exception raised by the API, something went wrong while retrieving referee state for sending extra info." << std::endl
+                  << msg << std::endl;
+    }
+
+	fs_msgs::ExtraInfo extra_info_msg;
+	extra_info_msg.doo_counter = state.doo_counter;
+	extra_info_msg.laps = state.laps.size();
```
Comment (SijmenHuizenga) [inline ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp:838]: still need to publish this message to `extra_info_pub`?

**Comment by SijmenHuizenga** [comment]
Thanks a lot for submitting this pr! Would you be able to add a line to the docs about this method as well?

**Comment by SijmenHuizenga** [comment]
Looks great!

Let's merge and ignore this github pipeline. It seems broken to something unrelated to this build.

**Comment by wouter-heerwegh** [comment]
> Looks great!
> 
> Let's ignore this github pipeline. It seems broken to something unrelated to this build.
> 
> After you confirm the location of the orange cone is correct we can merge 👍

The orange cones are detected correctly. It was however not symmetrical
image (1)
So I updated the blueprint

**Comment by SijmenHuizenga** [comment]
Cool! This seems ready to be merged. 

Commit #ba977f2 and #227d263 are already on master so these shouldn't be added again. On second thought updating the orange cone positions doesn't seem to relate to the topic of this pr, so let's leave it out.

I didn't want to force-push to this branch so I cherry-picked the commits that should be merged, added them to #261, and merged it.

**Pull request closed without merging**